diesel_cli/src/cli.rs

```diff
@@ -335,6 +335,14 @@ pub fn build_cli() -> Command {
                 .number_of_values(1)
                 .help("A list of derives to implement for every automatically generated SqlType in the schema, separated by commas."),
         )
+        .arg(
+            Arg::new("domains-as-custom-types")
```

Comment:
I would use the pg prefix here as well.

Reply:
[9ebddb4](https://github.com/diesel-rs/diesel/pull/4594/commits/9ebddb4d7d3052043690a2398dac6192af13d88d)